Builds.md

```diff
@@ -189,7 +189,11 @@ Build jobs for releases must be executed in proper order on the build server, i.
    * For both features select the `Version Range` property, open the version selection dialog, and click on the only _Available version_ to match exactly the new release version
 1. Publish websites
    * Remove `published: false` from release post
-   * [Create PR](https://github.com/eclipse/xtext/compare/website-published...website-master?expand=1) to merge branch `website-master` into `website-published` 
+   * [Create PR](https://github.com/eclipse/xtext/compare/website-published...website-master?expand=1) to merge branch `website-master` into `website-published`
+1. Update Marketplace entries
```

Comment:
@kthoms @pngowda is this correct? does the marketplace only support one version?
if not how do i create a new one?

Reply:
Marketplace supports multiple versions, but we did not use this feature so far. We should use that from now on. So please change the text to "Update Marketplace entries for [Xtext](https://marketplace.eclipse.org/content/eclipse-xtext/edit) and [Xtend](https://marketplace.eclipse.org/content/eclipse-xtend/edit). Add a new solution version."

![screenshot 85](https://user-images.githubusercontent.com/265597/51019940-2c4f4a80-157d-11e9-9517-dcad3e9de271.png)